src/ui/public/agg_types/metrics/lib/sibling_pipeline_agg_controller.js

```diff
@@ -0,0 +1,31 @@
+import safeMakeLabel from './safe_make_label';
+
+const siblingPipelineAggController = function (type) {
+  return function ($scope) {
+
+    $scope.aggType = type;
+    //$scope.aggParam = $scope.agg.params[type];
+    $scope.aggTitle = type === 'customMetric' ? 'Metric' : 'Bucket';
+    $scope.aggGroup = type === 'customMetric' ? 'metrics' : 'buckets';
+    $scope.safeMakeLabel = safeMakeLabel;
+
+    $scope.rejectAgg = function (agg) {
```

Comment:
you are right, this is not used here ... to much copy pasting from parent pipeline agg :)